File: Meals V1/app/src/main/java/org/techtown/project5/Network/retrofit/interfaces/School.java
package org.techtown.project5.Network.retrofit.interfaces;

import org.techtown.project5.Network.Data;
import org.techtown.project5.Network.response.Response;

import retrofit2.Call;
import retrofit2.http.GET;
import retrofit2.http.Query;

public interface School {

    @GET("/search-school")
    Call<Response<Data>> searchSchool(@Query("school_name") String SchoolName);

}

File: Meals_Android V2/app/src/main/java/org/techtown/project5/network/retrofit/interfaces/School.kt
package org.techtown.project5.network.retrofit.interfaces

import org.techtown.project5.network.Data
import org.techtown.project5.network.Response
import retrofit2.Call
import retrofit2.http.GET
import retrofit2.http.Query

interface School {

    @GET("search-school")
    fun searchSchool(@Query("school_name") schoolName : String) : Call<Response<Data>>
}